### ISSUE TITLE
A lot of warnings

### ISSUE THREAD

**Issue by stepanzh** [vvpisarev/Downhill.jl#26]
A lot of warnings come from line search (branch issue24)

```
┌ Warning: Function change is too small, using parabolic approximation
│   Δyp = -5.078140390443264e-9
│   Δy = -5.077506415318567e-9
└ @ Downhill ~/.julia/dev/Downhill/src/linesearch.jl:70
┌ Warning: Function change is too small, using parabolic approximation
│   Δyp = -4.152333226039501e-9
│   Δy = -4.15183620816606e-9
└ @ Downhill ~/.julia/dev/Downhill/src/linesearch.jl:70
┌ Warning: Function change is too small, using parabolic approximation
│   Δyp = -3.3793413183067366e-9
│   Δy = -3.3790215070439444e-9
└ @ Downhill ~/.julia/dev/Downhill/src/linesearch.jl:70
┌ Warning: Function change is too small, using parabolic approximation
│   Δyp = -2.7361828176819865e-9
│   Δy = -2.7360016474631266e-9
└ @ Downhill ~/.julia/dev/Downhill/src/linesearch.jl:70
```

**Comment by vvpisarev** [comment]
Must be fixed by #27

**Comment by vvpisarev** [comment]
Fixed by #27
